src/Process/3dTilesProcessing.js

```diff
@@ -120,21 +117,13 @@ function computeNodeSSE(camera, node) {
 }
 
 export function init3dTilesLayer(context, layer) {
-    if (layer.initialised) {
-        return;
-    }
-    layer.initialised = true;
-
-    Fetcher.json(layer.url, layer.networkOptions).then((tileset) => {
-        const urlPrefix = layer.url.slice(0, layer.url.lastIndexOf('/') + 1);
-        layer.tileIndex = new $3dTilesIndex(tileset, urlPrefix);
-        layer.asset = tileset.asset;
-        requestNewTile(context.view, context.scheduler, layer, tileset.root, undefined).then(
-            (tile) => {
-                layer.object3d.add(tile);
-                tile.updateMatrixWorld();
-                layer.root = tile;
-            });
+    layer.preprocessedPromise.then((layer) => {
```

Comment:
There's nothing preventing this code from being called multiple times (by this https://github.com/iTowns/itowns/blob/208fdad43c3553b283694db97b9f24e028c17a26/examples/3dtiles.html#L61). What are the implications of a promise being redefined multiple times?

More broadly, shouldn't the preprocessedPromise be completed before the layer starts updating?